### PR TITLE
I cant have external dependencies in diff_covers __init__.py

### PR DESCRIPTION
**Pull request by Bachmann1234**
setup.py imports it before any dependencies are resolved

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+12/11/2019 v2.5.0
+ 
+ * Diff quality now has a plugin system! From now own this project will avoid adding new tools and instead direct those to become plugins instead. Perhaps we can look into moving exisitng tools into separate plugins over time however, currently im leaving it allone. Thank you @barrywhart (https://github.com/Bachmann1234/diff_cover/pull/128)
```
Comment (nicoddemus) [inline CHANGELOG]: Typo: alone

File: README.rst [hunk 2]
```diff
@@ -317,7 +317,7 @@ The plugin implementation will look something like the example below. This is
 a simplified example based on a working plugin implementation.
 
 .. code:: python
-    import diff_cover
+    import diff_cover.hook import hookimpl as diff_cover_hookimpl
```
Comment (barrywhart) [inline README.rst]: This is not quite right. I tested on the SQLFluff plugin, and this is what we need:
```
from diff_cover.hook import hookimpl as diff_cover_hookimpl
```

**Comment by Bachmann1234** [comment]
oh doh! Good catch.
-Matt Bachmann

File: CHANGELOG (hunk 1 above)
Comment (Bachmann1234) [reply CHANGELOG]: There was a second typo as well! Thanks!

File: README.rst (hunk 2 above)
Comment (Bachmann1234) [reply README.rst]: Thanks for catching this!

**Comment by Bachmann1234** [comment]
Ok changes addressed. Any final thoughts before I merge and release this version @barrywhart ?

**Comment by barrywhart** [comment]
The PR looks good to me, @Bachmann1234!

**Comment by Bachmann1234** [comment]
Oh I know what the squirrel means. I will have this released tonight.

**Pull request merged**